optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/SubSingleStatistic.java

```diff
@@ -187,10 +200,10 @@ private void readCsvStatisticFile() {
     public void unhibernatePointList() {
         if (!getCsvFile().exists()) {
             throw new IllegalStateException("The csvFile ( " + getCsvFile() + " ) of the statistic ( " + getStatisticType()
-                    + " ) of the single benchmark ( " + singleBenchmarkResult + " ) doesn't exist.");
+                    + " ) of the single benchmark ( " + subSingleBenchmarkResult + " ) doesn't exist.");
```

Comment:
Consistency nitpicking: no spaces inside the braces:

    + " ) of the single benchmark ( " + subSingleBenchmarkResult + " ) doesn't exist.");

becomes

    + ") of the single benchmark (" + subSingleBenchmarkResult + ") doesn't exist.");